docs/block-api.md

```diff
@@ -127,36 +127,52 @@ useOnce: true,
 
 * **Type:** `Object`
 
-Optional block extended support features. The following options are supported, and should be specified as a boolean `true` or `false` value:
+Optional block extended support features. The following options are supported, and should be specified as a boolean `true` or `false` value in most of the cases:
+
+- `align` (default `false`): This property adds block controls which allow to change block's alignment.
+
+```js
+// Add the support for block's alignment.
+align: true,
+// Pick which alignment options to display.
+align: [ 'left', 'right', 'full' ],
+```
+
 
 - `anchor` (default `false`): Anchors let you link directly to a specific block on a page. This property adds a field to define an id for the block and a button to copy the direct link.
 
 ```js
-// Add the support for an anchor link.
+// Add the support for the anchor link.
 anchor: true,
 ```
 
 - `customClassName` (default `true`): This property adds a field to define a custom className for the block's wrapper.
 
 ```js
-// Remove the support for a the custom className .
+// Remove the support for the custom className .
 customClassName: false,
 ```
 
 - `className` (default `true`): By default, Gutenberg adds a class with the form `.wp-block-your-block-name` to the root element of your saved markup. This helps having a consistent mechanism for styling blocks that themes and plugins can rely on. If for whatever reason a class is not desired on the markup, this functionality can be disabled.
 
 ```js
-// Remove the support for a the generated className .
+// Remove the support for the generated className .
 className: false,
 ```
 
 - `html` (default `true`): By default, Gutenberg will allow a block's markup to be edited individually. To disable this behavior, set `html` to `false`.
 
 ```js
-// Remove support for an HTML mode.
+// Remove the support for the HTML mode.
 html: false,
 ```
 
+- `wideAlign` (default `true`): Gutenberg allows to enable [wide alignment](themes.md#wide-alignment) for your theme. To disable this behavior for a single block, set this flag to `false`.
```

Comment:
This link won't work when ported to the WordPress.org handbook. It needs to be an absolute URL.